### ISSUE TITLE
TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined

### ISSUE THREAD

**Issue by RamanBut-Husaim** [ionide/ionide-vscode-fsharp#327]
Hello,

When I try to open the [following folder](https://github.com/RamanBut-Husaim/FSharp-Practice/tree/master/FPinFSharp) in Ionide I receive the error that was taken from the developer tools:

```JavaScript
Uncaught Exception:  TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
extensionHost.ts:293[Extension Host] TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:2176:30
    at Object.<anonymous> (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1689:24)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1674:39
    at Object.<anonymous> (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1689:24)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1674:39
    at Object.<anonymous> (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1689:24)
    at Function.from (native)
    at mapResult (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:24:24)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:145:20
    at Array.map (native)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:144:22
    at handleNotification (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:147:7)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:170:9
    at WebSocket.<anonymous> (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Core/LanguageService.js:377:13)
    at emitTwo (events.js:106:13)
    at WebSocket.emit (events.js:191:7)A.logExtensionHostMessage @ extensionHost.ts:293
shell.ts:541Cannot read property 'Symbol(Symbol.iterator)' of undefined: TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:2176:30
    at Object.<anonymous> (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1689:24)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1674:39
    at Object.<anonymous> (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1689:24)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1674:39
    at Object.<anonymous> (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/node_modules/fable-core/fable-core.js:1689:24)
    at Function.from (native)
    at mapResult (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:24:24)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:145:20
    at Array.map (native)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:144:22
    at handleNotification (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:147:7)
    at /Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Components/Errors.js:170:9
    at WebSocket.<anonymous> (/Users/Rakkatakka/.vscode/extensions/Ionide.Ionide-fsharp-2.17.1/Core/LanguageService.js:377:13)
    at emitTwo (events.js:106:13)
    at WebSocket.emit (events.js:191:7)e.onUnexpectedError @ shell.ts:541
```

OS: macOS Sierra 10.12.2
Compiler: mono 4.8.0 (I tried the latest stable version as well)

```
Mono JIT compiler version 4.6.2 (mono-4.6.0-branch/08fd525 Thu Nov 10 20:28:28 EST 2016)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           normal
	SIGSEGV:       altstack
	Notification:  kqueue
	Architecture:  x86
	Disabled:      none
	Misc:          softdebug 
	LLVM:          yes(3.6.0svn-mono-master/8b1520c)
	GC:            sgen
```

```
Mono JIT compiler version 4.8.0 (mono-4.8.0-branch/f5fbc32 Mon Nov 14 14:10:00 EST 2016)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           normal
	SIGSEGV:       altstack
	Notification:  kqueue
	Architecture:  x86
	Disabled:      none
	Misc:          softdebug 
	LLVM:          yes(3.6.0svn-mono-master/8b1520c)
	GC:            sgen

```

It seems that due to this error there is no error highlighting on save.

Please, let me know if additional info is required.

Thanks.

**Comment by et1975** [comment]
This looks familiar, I think it was fixed in later fable.

**Comment by Krzysztof-Cieslak** [comment]
It's not fable error, I'm getting some wrongly handled message and I try to iterate over something that's not Array / Seq.

**Comment by Krzysztof-Cieslak** [comment]
Fixed in `2.18.2`

**Comment by RamanBut-Husaim** [comment]
@Krzysztof-Cieslak Thanks a lot!
